physical/raft/raft.go

```diff
@@ -28,8 +22,14 @@ import (
 	"github.com/hashicorp/go-uuid"
 	"github.com/hashicorp/raft"
 	autopilot "github.com/hashicorp/raft-autopilot"
+	"github.com/hashicorp/raft-boltdb/v2"
 	snapshot "github.com/hashicorp/raft-snapshot"
-	raftboltdb "github.com/hashicorp/vault/physical/raft/logstore"
```

Comment:
Are there files we can delete from there now?

Reply:
Yeah, I deleted that entire `logstore` directory, but git recorded it as renaming files because of the import of the shared raft-boltdb library. You can see it in the diff if you scroll all the way to the bottom and look at the vendor changes.